### ISSUE TITLE
Alternator open up Chrome Browser in endless Loop when asking for Auth Key

### ISSUE THREAD

**Issue by Caliban2017** [SamMackrill/gw2-alternator#153]
Hi,
i encountered a new Bug in the latest Version of Alternator, when the Game is asking for the Auth Key (i'm getting new IP-Ranges from my ISP, so GW2 is asking again to Authenticate the Accounts sometimes), my Chrome Browser is opening up in a endless Loop, even when i close all Browser Windows completly. It seems like Alternator is sending a Button Press / Shortcutkey in a endless Loop in this Moment, so it is impossible to user other Programs, because this spamming Keypress is blocking everything i try to do on Windows, only the "Stop!" Button in Alternator is helping here.

**Comment by SamMackrill** [comment]
> Hi, i encountered a new Bug in the latest Version of Alternator, when the Game is asking for the Auth Key (i'm getting new IP-Ranges from my ISP, so GW2 is asking again to Authenticate the Accounts sometimes), my Chrome Browser is opening up in a endless Loop, even when i close all Browser Windows completly. It seems like Alternator is sending a Button Press / Shortcutkey in a endless Loop in this Moment, so it is impossible to user other Programs, because this spamming Keypress is blocking everything i try to do on Windows, only the "Stop!" Button in Alternator is helping here.

Yes, sorry that is a feature I was forced to add last year and I don't like it:
![image](https://user-images.githubusercontent.com/84898/235454683-61fa8055-3e3e-4f87-a8d1-4d275c26fe7b.png)

I see your problem. I will have a think about that
